FoxHound.App/Blogs/Common/CommonBlogCommandValidator.cs

```diff
@@ -0,0 +1,18 @@
+﻿using FluentValidation;
+
+namespace FoxHound.App.Blogs.Common
+{
+    public class CommonBlogCommandValidator : AbstractValidator<IBlogCommand>
+    {
```

Comment:
You should create tests for this common validation and put the tests that were previously in the create/update blog commands here. This can now be tested on its own. The create/update validators now just test that this is being called. By having both tests in place, you can be sure that it's doing what you expect.

You can create the tests by creating a mock of IBlogCommand and using that to test this validator. I think we have some similar code in STS as well.

Reply:
Got it.